boards/chronos/include/periph_conf.h

```diff
@@ -26,9 +26,16 @@ extern "C" {
 #endif
 
 /**
- * @brief   Clock configuration
+* @todo
+*
+* @{
+* Move all clock configuration code here from the board.h
+* @}
+*/
+/**
+ * @name   Clock configuration
  *
- * @todo    Move all clock configuration code here from the board.h
```

Comment:
I tested adding /** @todo a todo line */ in between the `@name` block and the define. It works but the todo is then applied the define only not the while name block. What would you suggest ?